tests/functional/reporting-auth.js

```diff
@@ -39,6 +39,68 @@ define(
             assert.include(text, "Report as"); //Report as FooUser (logged in)
           })
           .end();
+      },
+
+      "Show double feature - buglist shows on focus, hides on blur": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://test.")
+          .sleep(500)
+          .end()
+          .findById("description")
+          .click()
+          .end()
+          .sleep(9500)
+          .findById("bugList")
+          .getVisibleText()
+          .then(function(text) {
+            assert.equal(text, "");
+          });
+      },
+
+      "Show double feature - 3 issues with url found": function() {
+        return FunctionalHelpers.openPage(
+          this,
+          url("/issues/new"),
+          ".wc-ReportForm-actions-button"
+        )
+          .findById("url")
+          .click()
+          .type("https://webctest.")
+          .sleep(500)
+          .end()
+          .findById("bugList")
+          .findAllByTagName("li a")
```

Comment:
I don't think `findAllByTagName` takes a CSS selector, but just a single `tagName`:

https://theintern.io/leadfoot/module-leadfoot_Element.html#findAllByTagName

(I could be wrong!)

If you want to filter with a selector, maybe `findAllByCssSelector` is what you want:

https://theintern.io/leadfoot/module-leadfoot_Element.html#findAllByCssSelector

Reply:
yes, perfect. thanks!